### PR TITLE
Replaces the Tad Heavy RR with the Mounted Flak Gun

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Replaces the basically entirely unused heavy RR with the ISG-15 Heavy Infantry Support Gun ™️ 
Caliber is 15cm (155mm)
Costs 1200 (Double of a TAT for reference, much less than the current Wombat) in req, shells are 100/200 (He/APFDS)points. Generally most of the cost is meant to be upfront rather than upkeep cost.

Has an 8 second windup to fire. 15 second fire delay and 8 second reload.
Shells function like normal bullets.
It fires literal howitzer shells (as in literally slightly more explosive radius, same travel speed of .75.)
HE Shells have unguided/guided forms, right click it with your other hand to change mode.
APFDS is 200/75, and pierces mobs and walls alike with ease. It has a screens worth of minimum range.
It is not IFF Capable.
Has TAT level scope.

Has TAT level Armor/Mobilty but slightly more integrity.
Basically only needs sprites for this to be finished.
## Why It's Good For The Game
The Heavy RR is basically pointless, you can't really make it good because double mortar kind of accomplishes the same task at a much more manageable cost and the req cost is insane for something that'll be forever stuck in Tad.
This basically accomplishes the idea of heavy direct-fire support weapon that heavy RR was attempting to accomplish in a slightly more mobile package, if unwieldy in windup and such.
## Changelog
:cl:
add: FK-88, a deployable direct fire support weapon that fires 15cm high explosive shells, replaces the RR-15.
del: Tad heavy RR has been replaced by the ISG-15 HISG.
/:cl:

### CODE REVIEW

**Comment by Tonklaatech** [comment]
Great another thing that shall never be bought,  kudos for the concept though.

**Comment by 74delta** [comment]
> Great another thing that shall never be bought, kudos for the concept though.

It's not tad only though. You can mount it like a regular TAT and drag it around. Unlike the Heavy RR which sucks and is only a tad mountable.

**Comment by Deroh-Ra** [comment]
Good PR

**Comment by yyzsong** [comment]
can this gun be rotated and can the firing be canceled by just stepping off of/unloading the gun?

File: code/modules/projectiles/mounted.dm [hunk 1]
```diff
@@ -24,6 +26,12 @@
 	else
 		icon_state = default_icon_state
 
+	if(has_anchored_sprite)
+		if(anchored)
+			icon_state = default_icon_state + "_anchored"
```
[the remark this answers is not in the document]
Comment (Pariah919) [reply code/modules/projectiles/mounted.dm:31]: There's no difference between E icons in the first place for deployed guns currently.

**Comment by Wirpy** [comment]
useless for tad without IFF lol espically for 1.2k points.

**Comment by Lumipharon** [comment]
bruh you didn't even remove it from maps.

**Pull request merged**